Cargo.toml

```diff
@@ -18,7 +18,7 @@ members = ["lorawan"]
 byteorder = "1"
 serde = { version = "1", features = ["rc", "derive"] }
 rust_decimal = { version = "1", features = ["serde-with-float"] }
-helium-proto = { git = "https://github.com/helium/proto", branch = "master", features = [
+helium-proto = { git = "https://github.com/helium/proto", rev = "d94ed4b", features = [
```

Comment:
Sorry what happens on master that breaks things here? I see another fix went in in a different downstream crate to work around things. May be easier to just adjust code?

Reply:
```
error[E0004]: non-exhaustive patterns: `Ok(helium_proto::services::router::envelope_down_v1::Data::PacketAck(_))` not covered
```

Reply:
looks like a new message exists

Reply:
we should be able to handle that though..

Reply:
Yea i'm just not sure what to do with it. I'm just trying to get this usable as a lib at the moment

Reply:
I believe it was added to HPR as part of https://github.com/helium/gateway-rs/pull/475

Reply:
I still believe the PR above should be fixed and merged up or this PR fixed to handle the case

Reply:
I would think be able to build at all would be a good enough PR on its own.

Updating is out of scope of what I can do at the moment

Reply:
I implemented a warning but there is also packet_ack_interval in PacketRouterRegisterV1

Reply:
that's ok, as long as it's not set I believe HPR will not issue regular acks